sbnanaobj/StandardRecord/SRTrueParticle.h

```diff
@@ -30,7 +30,7 @@ namespace caf
   class SRTrueParticle
     {
     public:
-      SRTrueParticlePlaneInfo plane[3];
+      std::vector<SRTrueParticlePlaneInfo> plane[3]; //!< Per-plane, per-cryostat deposition information
```

Comment:
I agree that the other order is more natural. Does the syntax you suggest work, and would it work with your plane flattening stuff?

If not, I also can see the merit of just having a two element array for the cryostat.

Reply:
I'm 90% sure that is correct C++ syntax, and 70% sure that it will work with srproxy. The test would be to include sbnana in your release too, and make sure compilation doesn't blow up in StandardRecord/Proxy/.